plugins/modules/_gluster_heal_facts.py

```diff
@@ -0,0 +1 @@
+gluster_heal_info.py
```

Comment:
is this file needed?

Reply:
_gluster_heal_facts.py was used to achieve the backward compatibility, as the gluster_heal_facts is renamed to gluster_heal_info. 
1. The user can still access the gluster_heal_facts module, however it is a symbolic link to gluster_heal_info. 
2. When gluster_heal_facts is provided as the module name , the ansible core library prefix the module name with '_' and accesses the _gluster_heal_facts module

However with the introduction of collections, it is required to mention module names in fully qualified name and the above process may not be relevant. Thus the _gluster_heal_facts is not needed and has been removed